ui/admin/src/assets/i18n/en.json

```diff
@@ -99,17 +99,20 @@
   "has 1 patron attached": "has 1 patron attached",
   "has # patrons attached": "has # patrons attached",
   "The default record cannot be deleted": "The default record cannot be deleted",
-  "The recording contains settings": "The recording contains settings",
+  "The recording contains settings": "The record contains settings",
```

Comment:
Should be changed in the original string?

Reply:
Ok